packages/theme/src/cli/services/dev.ts

```diff
@@ -84,10 +84,13 @@ export function renderLinks(store: string, themeId: string, host = DEFAULT_HOST,
       [
         {
           link: {
-            label: 'Share your theme preview',
+            label: `Share your theme preview`,
```

Comment:
I dont think you need to backtick here

Reply:
Oops, I merged the PR without applying the change. My apologies, @albchu. I've implemented this improvement here: https://github.com/Shopify/cli/pull/2996.